appinterface/projection/crossfire/view/crossfire_validators_stats.go

```diff
@@ -63,13 +63,42 @@ func (view *CrossfireValidatorsStats) FindBy(key string) (int64, error) {
 	return value, nil
 }
 
+func (view *CrossfireValidatorsStats) FindAllBy(key string) ([]CrossfireValidatorsStatsRow, error) {
```

Comment:
Maybe `FindAllLike` is more appropriate?

Reply:
Makes sense.